client/src/components/ElementEditor/Header.scss

```diff
@@ -0,0 +1,92 @@
+.element-editor {
```

Comment:
Ok can we please refactor this stylesheet a little so that the class names are more specific? I'm imagining that the "block" in BEM in this component is `element-editor-header` always, and the "element" would be things like "heading", "title", "icon" etc